src/System.Net.Security/tests/UnitTests/Fakes/FakeSslStream.Implementation.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace System.Net.Security
+{
+    public partial class SslStream
+    {
+        private void ValidateCreateContext(SslClientAuthenticationOptions sslClientAuthenticationOptions, RemoteCertValidationCallback remoteCallback, LocalCertSelectionCallback localCallback)
+        {
+            if(_shutdown == true && _nestedWrite == 0)
+            {
+
+            }
```

Comment:
?

Reply:
Ahh, because they weren't used in SslStream, or FakeSslStream.Implementation they stop the build due to being set but never used. The shutdown can now be removed due to the removal of the property, the nested write will need to be there still but I will comment.